packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import { observer } from 'mobx-react-lite';
 import { Text, Popover } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { isMobile } from '@deriv/shared';
 import BalanceText from 'Components/elements/text/balance-text';
-import { useStores } from 'Stores';
+// import { useStores } from 'Stores';
+import { observer, useStore } from '@deriv/stores';
 import './asset-summary.scss';
 import TotalAssetsLoader from 'Components/pre-loader/total-assets-loader';
 
-const AssetSummary = () => {
-    const { traders_hub, client, common } = useStores();
+const AssetSummary = observer(() => {
+    const { traders_hub, client, common }: any = useStore();
```

Comment:
```suggestion
    const { traders_hub, client, common, exchange_rates } = useStore();
```
@shayan-deriv Please add the types for those property which TS is complaining about, I've already added them [here](https://github.com/binary-com/deriv-app/pull/7634/files#diff-fdcfb06a6f9a508c26db37be5cbe82ace4cf3d3ec4a325a8e29274e35a2c6c54), I guess you can just use them 🙇🏻